views/file.pug

```diff
@@ -12,7 +12,7 @@ block content
         span.header-title
             span.octicon(class=icon)
             |  #{title}
-            if (search == null)
+            if search == null && lastModified
```

Comment:
This would be useful to have but will take a couple changes to make it dynamic for static pages. So I'd like to skip rendering it for now and instead open a bug after we get this merged. Then I'll fix the bug.

How do you feel about that?

Reply:
Definitely! That was in my (long) list of things to change, and yes this should definitely be done in a separate PR, for now static rendering without search is already a huge step forward 😃 

FWIW, I had in mind at some point to generate a static search index using http://elasticlunr.com, to use offline for statically generated version.

Reply:
Noted. I'll try to pick up work on this in the next month or so.

Reply:
https://github.com/sinedied/hads/issues/49

Reply:
Oops, I thought you were referring to search in static html, my bad 😅 
For the last modified tag, a simple trick for static HTML could simply be to print the date directly instead of "X time ago", so it would not need to be dynamic.

Reply:
That would work. But, if we go that route, we should change the server to display dates in the same format.